src/pages/[category]/[...slug].astro

```diff
@@ -133,3 +153,25 @@ categoryPosts.sort((a, b) => {
     </div>
   </main>
 </Layout>
+
+<style is:global>
+  .prose
```

Comment:
If a `p` is the first child of the rendered Markdown article content it's turned into a lead paragraph – a bit fragile but imo works for now: 
<img width="1327" alt="image" src="https://github.com/user-attachments/assets/f995a3c3-4aa9-4e39-96eb-6691249f793f" />